tests/migrations/test_commands.py

```diff
@@ -1070,3 +1070,35 @@ def test_ticket_23799_squashmigrations_no_optimize(self):
             call_command("squashmigrations", "migrations", "0002",
                          interactive=False, verbosity=1, no_optimize=True, stdout=out)
         self.assertIn("Skipping optimization", force_text(out.getvalue()))
+
+    def test_squashmigrations_valid_start(self):
+        """
+        Tests that squashmigrations accepts a starting migration.
```

Comment:
Considering which cases I overlooked when writing the test I kind of see why :wink: Thanks!